scripts/start-utils

```diff
@@ -109,10 +117,86 @@ function log() {
   if isDebugging || isTrue "${LOG_TIMESTAMP:-false}"; then
     ts=" $(date --rfc-3339=seconds)"
   fi
-  echo "[init]${ts} $*"
+  echo -e "[init]${ts} $*"
   eval "$oldState"
 }
 
+function getCommandFromDistro(){
+  distro=$(getDistro)
+  command=
+  if [[ $distro == alpine ]]; then
+    command="su-exec"
+  else
+    command="gosu"
+  fi
+  echo $command
+}
+
+function getErrorColoredLogString() {
+  echo "\033[0;31m[ERROR] $* \033[0m"
+}
+
+function getWarningColoredLogString() {
+  echo "\033[0;33m[WARN] $* \033[0m"
+}
+
+function error() {
+  echo -e $(getErrorColoredLogString "$*")
+}
+
+function logError() {
+  log $(getErrorColoredLogString "$*")
+}
+
```

Comment:
Why both `error` and `logError`? I'd say go with `logError` and just collapse the color coding into it.

Same for warning.

Reply:
it's for those file : /bin/mc-send-to-console
So they can use the new log system without using the log function that add [init] before all logged lines

Reply:
Thanks, I forgot that the logs get prefixed with `[init]`. In that case, they're fine as is.